docs/gettinginvolved/infrastructure/release-process.rst

```diff
@@ -172,12 +172,18 @@ This review step is to ensure that all acceptance tests pass on the release bran
 .. warning:: This process requires ``Vagrant`` and should be performed on your own workstation;
             **not** on a :doc:`Flocker development machine <vagrant>`.
 
+#. Export the version number of the release being reviewed as an environment variable for later use:
+
+   .. code-block:: console
+
+      export VERSION=0.1.2
+
 #. Do the acceptance tests:
 
    - Add the tutorial vagrant box that BuildBot has created from the release branch.
 
      The URL can be found by examining the "upload-base-box" step of the ``flocker-vagrant-tutorial-box`` builder.
```

Comment:
The URL cannot be found by examining the "upload-base-box" step of the ``flocker-vagrant-tutorial-box`` builder, so this should either be removed, or changed to reference the "tutorial box" link under ``write-base-box-metadata``.

Or to clarify what I was suggesting on the release PR, this line and the line below ("The URL will look like...") can be removed, and the console block below can be changed from:
```
.. code-block:: console

        vagrant box add <URL>
```

to

```
.. code-block:: console

        vagrant box add http://build.clusterhq.com/results/vagrant/release/flocker-${VERSION}/flocker-tutorial.json
```

I think that this is the best solution.